lively.morphic/serialization.js

```diff
@@ -81,8 +82,27 @@ export async function saveWorldToResource (world = MorphicEnv.default().world, t
   } finally { i && i.remove(); }
 }
 
-export function copyMorph (morph) {
-  return deserializeMorph(serializeMorph(morph), { migrations, reinitializeIds: true });
+export function copyMorph (morph, realCopy = false) {
+  if (!realCopy) {
+    return deserializeMorph(serializeMorph(morph), { migrations, reinitializeIds: true });
```

Comment:
```suggestion
    return deserializeMorph(serializeMorph(morph), { migrations, reinitializeIds: false });
```
Thought we didn't want to do this?

Reply:
We are not sure what you mean exactly, but the values of reinitializeIds have not been changed by us. Since everything worked (except the comments) before our changes, we see no reason to change this part.